smoketest/src/main/java/com/mongodb/jdbc/smoketest/SmokeTest.java

```diff
@@ -0,0 +1,70 @@
+package com.mongodb.jdbc.smoketest;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+/**
+ * SmokeTest runs a test on built artifacts to verify that connection,
+ * metadata retrieval, and querying is successful
+ */
+public class SmokeTest {
+    static final String URL = "jdbc:mongodb://localhost";
+    static final String DB = "integration_test";
+    public static final String MONGOSQL = "mongosql";
+
+    public static Connection getBasicConnection(String url, String db)
+            throws SQLException {
+        Properties p = new java.util.Properties();
+        p.setProperty("dialect", MONGOSQL);
+        p.setProperty("user", System.getenv("ADL_TEST_LOCAL_USER"));
+        p.setProperty("password", System.getenv("ADL_TEST_LOCAL_PWD"));
+        p.setProperty("authSource", System.getenv("ADL_TEST_LOCAL_AUTH_DB"));
+        p.setProperty("ssl", "false");
+        p.setProperty("database", db);
+        return DriverManager.getConnection(URL, p);
+    }
+
+    public static void databaseMetadataTest(DatabaseMetaData dbMetadata) throws SQLException {
+        ResultSet rs = dbMetadata.getColumns(null, "%", "%", "%");
```

Comment:
should we add smoke-test-specific data to the `resources/integration-test/testdata` directory so we can safely edit the existing data as needed for integration tests without breaking these tests? This also applies to the `queryTest` method below which queries `class` and expects 5 rows.

Reply:
That's seems a little bit overkill, no?
To be fair, we probably don't need to check the actual row count, making sure that it's over 1 let us check that we can retrieve data and in this case even exhaust a resultset without problem. Smoke test should be light, just making sure the produced lib has no issues and can load/be used with no errors.

Reply:
Checking that it is just greater than 1 is a good idea. I just don't want these tests to be tied so specifically to data that isn't strictly for them, but just ensuring the result set is non-empty is a great way of doing that without introducing a new dataset.

Reply:
I was debating on whether to put a greater than equals when I was initially writing this.   A greater than 1 sounds good.  I'll make that change.